pennylane_lightning/core/_serialize.py

```diff
@@ -85,6 +85,20 @@ def __init__(self, device_name, use_csingle: bool = False):
         self.hamiltonian_c64 = lightning_ops.observables.HamiltonianC64
         self.hamiltonian_c128 = lightning_ops.observables.HamiltonianC128
 
+        self.use_mpi = False
+
+        if use_mpi:
+            self.use_mpi = use_mpi
+            self.statevectormpi_c128 = lightning_ops.StateVectorMPIC128
+            self.named_obsmpi_c64 = lightning_ops.observablesMPI.NamedObsMPIC64
+            self.named_obsmpi_c128 = lightning_ops.observablesMPI.NamedObsMPIC128
+            self.hermitian_obsmpi_c64 = lightning_ops.observablesMPI.HermitianObsMPIC64
+            self.hermitian_obsmpi_c128 = lightning_ops.observablesMPI.HermitianObsMPIC128
+            self.tensor_prod_obsmpi_c64 = lightning_ops.observablesMPI.TensorProdObsMPIC64
+            self.tensor_prod_obsmpi_c128 = lightning_ops.observablesMPI.TensorProdObsMPIC128
+            self.hamiltonianmpi_c64 = lightning_ops.observablesMPI.HamiltonianMPIC64
+            self.hamiltonianmpi_c128 = lightning_ops.observablesMPI.HamiltonianMPIC128
```

Comment:
Do we need separate names for all of these? Can we not overwrite the names from above, or do we require both?

Reply:
yes, we need separate names for now since we plan to support both single node and distributed backends once LGPU-MPI is installed. We have to use different names to identify them.